### ISSUE TITLE
UnicodeDecodeError when installing without locales set

### ISSUE THREAD

**Issue by maingoh** [modlinltd/django-advanced-filters#95]
With pip3 and without locales set I get the following errors:
```
Collecting django-advanced-filters==1.1.0 (from -r deploy/docker-base/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/45/68/98aed982c7a56eb47dfed7cf47703c533b7bdc4ea65f85bae7fbf98c48eb/django-advanced-filters-1.1.0.tar.gz (72kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-o9zraagc/django-advanced-filters/setup.py", line 62, in <module>
        long_description=get_full_description(),
      File "/tmp/pip-build-o9zraagc/django-advanced-filters/setup.py", line 40, in get_full_description
        CHANGELOG = changelog.read()
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1016: ordinal not in range(128)
```
I need to do the following to fix it:
```
locale-gen en_US.UTF-8
export LANG=en_US.UTF-8 LANGUAGE=en_US.en LC_ALL=en_US.UTF-8
```

Can't we do something to avoid it ? It seems to be only because of the CHANGELOG.

**Comment by kut** [comment]
Same here!

**Comment by maingoh** [comment]
Fixed in #94
